lib/email-client.js

```diff
@@ -12,30 +12,6 @@ export default class EmailClient {
 		this.mailbox = new Mailosaur.Mailbox( mailboxId );
 	}
 
-	deleteAllEmailByID( emailID ) {
```

Comment:
This method is used here https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-log-in-out-spec.js#L248

Reply:
Also here:
https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-log-in-out-spec.js#L323
https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-log-in-out-spec.js#L390

Reply:
Also here:
https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-log-in-out-spec.js#L323
https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-log-in-out-spec.js#L390